app/javascript/youtube.js

```diff
@@ -0,0 +1,34 @@
+const async = require('async')
+
+window.onYouTubeIframeAPIReady = () => {
```

Comment:
I went ahead and moved the YouTube API shim to the bottom of `<body>`. I don't think there should be a load issue, but I'm willing to discuss it.

Reply:
I'm not 100% on the loading thing, it's just that turbolinks and DOM related things do not play well together usually and we've had problems with it in the past.

Reply:
I moved the API JS to the `<head>` tag along side the Twitch JS and then tied in to the Turbolinks events so that things should play nicely with partial page loads and not do stuff more than once.